pkg/TIPA.py

```
@@ -164,587 +165,498 @@ def __init__(self, img, mouse_pos, display_info_init, gui_queue):
+        wordlist = text.strip().split()
+        
+        if self.debug_mode >= 1:
+            logging.info(wordlist, wordlist[len(wordlist)-1])
+
+        new_wordlist = []
+        for word in wordlist:
+            if len(word) > 1 and re.match(r"^[-\(\)/.,\"\'a-zA-Z0-9_]*$", word):
+                new_wordlist.append(word.strip(r"[-'”\".`@_!#$%^&*<>?/\}{~:]"))
+        return new_wordlist
+
+    def process_image(self, attempt, lock, temp_files, is_inventory):
```

Ruff (0.8.0)

`process_image` is too complex (11 > 10)

(C901)